ports/windows/README.md

```diff
@@ -3,46 +3,73 @@ It is based on Unix port, and expected to remain so.
 The port requires additional testing, debugging, and patches. Please
 consider to contribute.
 
+All gcc-based builds use the gcc compiler from [Mingw-w64](mingw-w64.org),
+which is the advancement of the original mingw project. The latter is
+getting obsolete and is not actively supported by MicroPython.
+
+Build instruction assume you're in the ports/windows directory.
 
 Building on Debian/Ubuntu Linux system
 ---------------------------------------
 
-    sudo apt-get install gcc-mingw-w64
+    sudo apt-get install python3 build-essential gcc-mingw-w64
+    make -C ../../mpy-cross CROSS_COMPILE=i686-w64-mingw32-
```

Comment:
Is that correct?  Doesn't that build mpy-cross for the Windows system, yet in the following line mpy-cross is needed on the host Linux system to build the main Windows MicroPython?  So it should be `make -C ../../mpy-cross`?

Reply:
Yes, good point. I didn't test this particular case though so first let me make sure it actually works.

Reply:
Ok I tested a lot of things yesterday an on second thought, this one as well; but where this would not work on a normal linux system I was testing this on WSL, so even though the generated mpy-cross executable was a windows executable WSL shells can run those. Anyway, fixed now, and tested.